modules/core/src/lib/attribute/attribute-transition-utils.ts

```diff
@@ -120,6 +120,7 @@ export function getFloat32VertexFormat(size: 1 | 2 | 3 | 4): LumaVertexFormat {
       return 'float32x3';
     case 4:
       return 'float32x4';
+    default:
```

Comment:
Move throw to default clause for consistency?

Reply:
Done ✅